Scala doc update (sliding method)

BartoszJanota opened the pull request:
Added info about default step set to 1. Reading this doc (having no code) can be ambiguous.

BartoszJanota committed:
Scala doc update (sliding method)

Added info about default step set to 1. Reading this doc (having no code) can be ambiguous.

retronym commented:
Thanks for the contribution.

I'd suggest to also remove the "(defaults to 1)" from the overloaded version that accepts the step parameter.

Would you mind signing our CLA?

http://www.typesafe.com/contribute/cla/scala

BartoszJanota committed:
updated scala doc (overloaded sliding method)

Step is a method parameter and must be always given explicit, so info about default value is wrong.

BartoszJanota commented:
Sure, I'v just signed your CLA.

Thanks for suggesting, I corrected the overloaded version and made both docs clear.

BartoszJanota commented:
@retronym, I can't see why this branch has failed 2 checks after the 2nd commit, any suggestions?

retronym commented:
@BartoszJanota The failure in the scala-2.11.x-integrate-ide is a rencently fixed problem in our testing infrastructure on the 2.12.x branch. Rebasing your branch on the latest origin/2.12.x would resolve this.

However, for a documentation change like this I would suggest to target the 2.11.x branch. That would mean the improvement could be included in 2.11.6 (in a month or so) rather than in 2.12.0 (which is about a year away).

Here are the git commands you'd need:

```
# Close this PR
% git fetch origin
% git checkout patch-1
% git reset origin/2.11.x
% git cherry-pick  8a51e01
% git push -f
# Open a new PR targetting 2.11.x
```

BartoszJanota commented:
@retronym, thanks for your advice and help. I changed the target branch and opened a new PR.
